helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -172,6 +175,13 @@ exporters:
   # If collector is disabled, metrics and traces will be set to to SignalFx backend
   {{- include "splunk-otel-collector.otelSapmExporter" . | nindent 2 }}
   {{- end }}
+
+  splunk_hec:
+    endpoint: {{ include "splunk-otel-collector.logUrl" . }}
+    token: "${SPLUNK_HEC_TOKEN}"
```

Comment:
Why do you think it must be different env var instead of reusing `SPLUNK_ACCESS_TOKEN`?

Reply:
Yep I think we can get rid of SPLUNK_HEC_TOKEN, will add to list.

Reply:
Will you update it in this PR?

Reply:
No, followup PR as it's unrelated to this change. I'm trying very hard to limit scope of changes.